Configurations.md

```diff
@@ -1815,3 +1815,97 @@ What Write Mode to use when none is supplied: Replace, Overwrite, Display, Diff,
 - **Default value**: `"Overwrite"`
 - **Possible values**: `"Checkstyle"`, `"Coverage"`, `"Diff"`, `"Display"`, `"Overwrite"`, `"Plain"`, `"Replace"`
 - **Stable**: No
+
+## `blank_lines_upper_bound`
+
+Maximum number of blank lines which can be put between items. If more than this number of consecutive empty 
+lines are found, they are trimmed down to match this integer.
+
+- **Default value**: `1`
+- **Possible values**: *unsigned integer*
+- **Stable**: Yes
+
+### Example
+Original Code:
+
+```rust
```

Comment:
This code block will break the unmerged test in #2292 because that test expects all Rust code blocks to be preceded by "## \`CONFIG_NAME\`" and "#### \'CONFIG_VALUE\`", in that order. That's the convention that has been followed to this point, at least.

So we need to either update this documentation to match that convention or I need to update the test in #2292 to have a way to opt Rust code blocks out of verification. Is it feasible to modify this documentation or is the example code before the formatted code preferable for clarity?

Reply:
Mh, I think that these two configurations are different from the other ones in that changing the value is not "bijective": formatting with a decreased blank line bound will lose information, so increasing the blank line bound and formatting it again won't result in the code we started with. So I wouldn't use the convention here, as it would be less clear to the reader.

I am not sure if there should be a special case in your tests for this; it's probably to complicated right now. So you are probably right: opting out of the test for the first code block and still testing the others is probably fine. 

But I'm fine with whatever you (all) decide on :)